index.js

```diff
@@ -0,0 +1,47 @@
+let fs = require('fs');
+let solc = require('solc');
+let path = require('path');
+
+let contractCache;
+
+let WeTrustContract = {
+  /**
+   * Call this function upon server start. It reads and compiles the contract, and saves it in a cache variable.
+   * Throws if there was an error.
+   */
+  init: function(done) {
```

Comment:
I would suggest move this init() and execute it outside of WeTrustContract scope.  Basically, WeTrustContract would only have getContract() for now.